src/components/Calculator.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import './Calculator.css';
+
+const Calculator = () => {
+  const [result, setResult] = useState('0');
+
+  const handleClick = (value) => {
+    setResult((prevResult) => {
+      if (prevResult === '0') {
+        return value;
+      }
+      return prevResult + value;
+    });
+  };
+
+  const handleCalculate = () => {
+    try {
+    //   eslint-disable-next-line
```

Comment:
- [ ] Following the best practices, you are not meant to use any linters disable rule on your code `unless otherwise stated`. I'll suggest you remove this diable rule. :+1: